api/issue.go

```diff
@@ -133,6 +133,8 @@ type CreateDatabaseContext struct {
 	InstanceID int `json:"instanceId"`
 	// DatabaseName is the name of the database.
 	DatabaseName string `json:"databaseName"`
+	// TableName is the name of the table, if it is not empty, Bytebase should create a table after creating the database.
```

Comment:
Need to comment why there would be such a case to create table right after creating database

Reply:
Done.